### PR TITLE
Feature/add dynamic phone ip

### PR DESCRIPTION
**Pull request by saigon**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- 
-

### CODE REVIEW

**Comment by saigon** [comment]
Verified on local box in reserve, release, get-available nodes

**Review by gilkal — COMMENTED** [review]
@saigon Could you run flake8 and run_repo_tests and check the check boxes

**Review by gilkal — COMMENTED** [review]
Would you be able to add a description ,so it's slightly easier to understand? :)

**Comment by saigon** [comment]
Ran flake8 and run_repo_tests.sh

**Comment by saigon** [comment]
This changes will let the script pull the phone IP from server_pools to insert to job when the job running, not on job configure setup.

File: utilities/mobile_server_pool.py [hunk 1]
```diff
@@ -34,15 +36,27 @@ def get_nodes_available_from_mobile_pool(nodes_os_type, node_os_version):
...
     for row in query:
-        count = row["$1"]
+        doc_id = row["id"]
+        if nodes_os_type in MOBILE_OS:
+            check_alive = check_device_alive(doc_id)
+        else:
+            check_alive = check_vm_alive(doc_id)
+        if check_alive:
+            count += 1
     print(count)
```
Comment (gilkal) [inline utilities/mobile_server_pool.py:55]: Do you return the list of IPs? I see a print of the count, but no value returned

File: utilities/mobile_server_pool.py [hunk 2]
```diff
@@ -52,22 +66,33 @@ def get_nodes_from_pool_server(num_of_nodes, nodes_os_type, node_os_version, job
...
-            query = cluster.query(query_str)
-        if is_node_reserved and vm_alive:
-            pool_list.append(str(doc_id))
-
+        if nodes_os_type not in MOBILE_OS:
+            vm_alive = check_vm_alive(doc_id)
+            if not vm_alive:
+                query_str = "update `{}` set state=\"VM_NOT_ALLIVE\" where meta().id='{}' " \
+                    "and state='available'".format(doc_id, doc_id)
+                query = cluster.query(query_str)
```
Comment (gilkal) [inline utilities/mobile_server_pool.py:85]: Is query used anywhere? If not, maybe the "if not vm_alive" block could be removed?

File: utilities/mobile_server_pool.py [hunk 3]
```diff
@@ -52,22 +66,33 @@ def get_nodes_from_pool_server(num_of_nodes, nodes_os_type, node_os_version, job
...
+            if is_node_reserved and vm_alive:
+                pool_list.append(str(doc_id))
+        else:
+            device_alive = check_device_alive(doc_id)
+            if not device_alive:
+                query_str = "update `{}` set state=\"device_down\" where meta().id='{}' " \
+                            "and state='available'".format(doc_id, doc_id)
+                query = cluster.query(query_str)
```
Comment (gilkal) [inline utilities/mobile_server_pool.py:93]: Same as above, do we need this bit?

File: utilities/mobile_server_pool.py (hunk 1 above)
Comment (saigon) [reply utilities/mobile_server_pool.py:52]: No, it returns only number of available vm and could remote access to it.

**Review by gilkal — APPROVED** [review]

**Pull request merged**